modules/core/04-channel/client/cli/tx.go

```diff
@@ -0,0 +1,47 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+)
+
+// NewPruneAcknowledgementsTxCmd returns the command to create a new MsgPruneAcknowledgements transaction
+func NewPruneAcknowledgementsTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "prune-acknowledgements [port] [channel] [limit]",
+		Short: "Prune expired packet acknowledgements stored in IBC state",
+		Long: `Prune expired packet acknowledgements and receipts stored in IBC state. Packet ackwnowledgements and 
+		receipts are considered expired if a channel has been upgraded and the start and end pruning sequences have been set.`,
```

Comment:
whoops. in reference to this. Does it make sense to mention start/end pruning sequence. I'd think the only requirement here (which encapsulates start/end seq) is that channel has done upgrade, a nit to start the year.

Reply:
Yeah, I'd tend to agree with you. I just kind of wrote the long desc on autopilot to be honest. I can update it to remove the start/end pruning sequences